src/js/components/AppPageComponent.jsx

```diff
@@ -15,27 +23,23 @@ var tabsTemplate = [
   {id: "apps/:appid/configuration", text: "Configuration"}
 ];
 
+var statusNameMapping = {};
+statusNameMapping[AppStatus.RUNNING] = "Running";
+statusNameMapping[AppStatus.DEPLOYING] = "Deploying";
+statusNameMapping[AppStatus.SUSPENDED] = "Suspended";
+
```

Comment:
I think in ES6 we could write

```
var statusNameMapping = {
    [AppStatus.RUNNING]: "Running",
    [AppStatus.DEPLOYING]: "Deploying",
    [AppStatus.SUSPENDED]: "Suspended"
};
```
See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#Syntax